docs/src/components/TailwindClassnames/index.tsx

```diff
@@ -104,6 +122,12 @@ const TailwindClassnames = () => {
             <Heading type="title2" spaceAfter="normal">
               {group}
             </Heading>
+            {group === "Colors" && (
+              <Text type="info" weight="medium" spaceAfter="medium">
```

Comment:
Just a 💅 but since we have `spaceAfter` deprecated, can we use the `margin` prop instead here? One less place to change after we finally remove `spaceAfter`. WDYT?

Reply:
Actually I tried that and found a bug, that only in case of providing a px value it applies the margin. If I add something like *margin={{bottom: medium}}* -> nothing happens

Reply:
I hope the bug is not present in TW branch